src/components/KCheckbox/KCheckbox.vue

```diff
@@ -18,13 +18,15 @@
         v-if="modelValue"
         class="checkbox-icon"
         data-testid="check-icon"
+        decorative
```

Comment:
question: for checkbox & radio, is there anything in the DOM that declares the "checked" status other than this icon?